core/trino-main/src/main/java/io/trino/execution/CreateTableTask.java

```diff
@@ -194,7 +194,7 @@ ListenableFuture<Void> internalExecute(CreateTable statement, Session session, L
 
```

Comment:
Can we re-word the commit as `Allow connector to automatically coerce type during table creation`